core-web/libs/sdk/angular/README.md

```diff
@@ -52,9 +41,326 @@ const DOTCMS_CLIENT_CONFIG: ClientConfig = {
+| `customRenderers` | `CustomRenderers`     | ❌        | Custom rendering functions for specific blocks   |
+| `className`       | `string`             | ❌        | CSS class to apply to the container              |
+| `style`           | `CSSProperties`       | ❌        | Inline styles for the container                  |
+
+**Usage**:
+
+```typescript
+<dotcms-block-editor-renderer
+    [blocks]="contentlet.blockField"
+    [customRenderers]="myCustomRenderers"
+/>
+```
+
+
+**Editor Integration**:
+
+* Renders exactly as configured in dotCMS editor
+* Supports custom renderers for block types
+
+**Common Issues**:
+
+* Should not be used with DotCMSEditableText
+* Inherits styling from parent containers, which may cause conflicts
+* `DotCMSBlockEditorRenderer` only works with Block Editor fields. For other fields, use [`DotCMSEditableText`](#dotcmseditabletext).
+
+📘 For advanced examples, customization options, and best practices, refer to the [DotCMSBlockEditorRenderer README](https://github.com/dotCMS/core/tree/master/core-web/libs/sdk/angular/src/lib/components/DotCMSBlockEditorRenderer).
+
+### Directives
+
+#### DotCMSShowWhen
+
+**Overview**: Directive for conditionally showing content based on the current UVE mode.
+
+**Input**:
+
+| Input    | Type       | Required | Description                                    |
+|----------|------------|----------|------------------------------------------------|
+| `when`   | `UVE_MODE` | ✅        | The UVE mode when content should be displayed  |
+
+**Usage**:
+
+```typescript
+<div *dotCMSShowWhen="UVE_MODE.EDIT">
+    Only visible in edit mode
+</div>
+```
+
+**Editor Integration**:
+
+* Useful for mode-based behaviors outside of render logic
+
+**Common Issues**:
+
+* Requires proper mode enum from `@dotcms/types`
+
+### Services
+
+#### DotCMSEditablePageService
+
+**Overview**: Service for managing editable page state and interactions with the Universal Visual Editor.
+
+**Methods**:
+
+| Method                  | Parameters                    | Returns              | Description                                        |
+|------------------------|-------------------------------|---------------------|----------------------------------------------------|
+| `init`                 | `pageResponse: PageResponse`  | `void`             | Initializes the editable page service              |
+| `getPageAsset`         | -                            | `DotCMSPageAsset`   | Returns the current page asset                     |
+| `updatePageAsset`      | `asset: DotCMSPageAsset`     | `void`             | Updates the current page asset                     |
+| `isEditMode`           | -                            | `boolean`           | Checks if the page is in edit mode                 |
+
+**Usage**:
+
+```typescript
+@Component({...})
+export class PageComponent {
+    constructor(private editablePageService: DotCMSEditablePageService) {
+        this.editablePageService.init(pageResponse);
```

Comment:
the method is called `listen` and you need to subscribe to it

Reply:
Done!